Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java

```diff
@@ -249,6 +249,7 @@ public WebApp updateWebAppOnLinux(String sid, String webAppId, ImageSetting imag
         } else {
             // TODO: other types of ImageSetting, e.g. Docker Hub
         }
+        restartWebApp(sid, webAppId);
```

Comment:
not work for stopped web app. should call startWebApp according to the inner state